doc/configuration.rst

```diff
@@ -91,25 +88,25 @@ Security
 --------
 
 It is a good practice to have the cookies require a https connection for
-security reason. However, while developing this can prevent the authentication
+reasons of security. However, while developing, this can prevent the authentication
 from working. So by default this is turned off to provide an out-of-the-box
 working configuration, however you will want to change it in production.
 
-The setting to change is ``SESSION_COOKIE_SECURE``.
+The setting is ``SESSION_COOKIE_SECURE``.
 
 **Default** ``SESSION_COOKIE_SECURE =  False``
 
-To change to ``SESSION_COOKIE_SECURE = True``.
+Change this ``SESSION_COOKIE_SECURE = True`` when using the application in a production environment.
```

Comment:
Can you clarify here, @pypingou?  Just being "not sure" doesn't give @vatsala much information to work on.